src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1156,6 +1160,15 @@ protected void addWhitelistWarningsGuard(
   }
 
   @Override
+  protected void checkModuleName(String name)
+      throws FlagUsageException {
+    if (!TokenStream.isJSIdentifier(
+        extraModuleNameChars.matcher(name).replaceAll("_"))) {
```

Comment:
This looks like it would change "foo--bar.js" (with multiple -'s) to foo_bar.js (with only one _). Should the regex be `[-.]` instead of `[-.]+`?